### PR TITLE
Add "Pack" command on project context menu and top level build menu

### PR DESCRIPTION
**Pull request by mavasani**
The command implementation is currently a no-op. I am sending this PR uprfront so the localizable strings are in place before the localization freeze.

### CODE REVIEW

**Comment by mavasani** [comment]
Tagging @dotnet/project-system for review

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractGenerateNuGetPackageCommand.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                GetCommandStatusResult.Handled(GetCommandText(), CommandStatus.Enabled) :
+                GetCommandStatusResult.Unhandled;
+
+        protected override async Task<bool> TryHandleCommandAsync(IProjectTree node, bool focused, Int64 commandExecuteOptions, IntPtr variantArgIn, IntPtr variantArgOut)
+        {
+            if (!ShouldHandle(node)) return false;
+
+            await _threadingService.SwitchToUIThread();
+
+            // TODO: Generate NuGet package.
```
Comment (333fred) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractGenerateNuGetPackageCommand.cs]: Uh, is this actually ready for review?

Comment (mavasani) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractGenerateNuGetPackageCommand.cs]: Yes, we have the Loc freeze this Friday, so Sri asked me to get all the UI pieces in place before implementing the command. I hope go get the command implementation done soon.

Comment (333fred) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractGenerateNuGetPackageCommand.cs]: Ah, ok. In that case, can you remove the AppliesTo attributes below, so the command doesn't show up in the context menu until it's actually implemented?

Comment (mavasani) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractGenerateNuGetPackageCommand.cs]: Sure, done.

**Comment by srivatsn** [comment]
I'm going back and forth on whether the command should be called Pack or Package. Pack is consistent with the cli. So that's fine I guess. @davidfowl @damianedwards what did we call the command in xproj?

**Comment by mavasani** [comment]
I need to merge this in ASAP for Loc Freeze. If I get no pushbacks by EOD, I am going to merge it in.

**Comment by nguerrera** [comment]
Can we please merge sooner than EOD? Handoff is first thing in the morning on Monday and I need to run the converter beforehand.

**Comment by mavasani** [comment]
Sure, I'll merge it now. Note that I have separate PR for the actual Package property page which has lot of stuff to be localized. However, I won't be likely able to merge it today due to pending reviews. Is Monday fine for that change?

**Comment by davidfowl** [comment]
Where the UI screenshot?

**Pull request merged**